components/TProfile.vue

```diff
@@ -143,6 +138,11 @@
       class="w-full mt-4"
     />
 
+    <TCommunity
```

Comment:
Make sure you rebase your code on top of `main` branch.
This code change has nothing to do with your task. Make sure you don't break things.